commands/serverless.js

```diff
@@ -10,6 +10,8 @@ module.exports = function (input, flags, logger, options) {
   const cwd = options.cwd
   const out = flags.out
   const env = flags.env
+  const vpcSecurityGroups = flags.vpcSecurityGroups
+  const vpcSubnets = flags.vpcSubnets
```

Comment:
I am aware your second example will not work, however, I was under the impression that the following works:

```
node ./bin/index.js serverless --vpc-subnets="space, after, each, comma"
```

I may have this wrong though...